rcvSpec.yaml

```diff
@@ -0,0 +1,367 @@
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/PollDetails'
+      security:
+        - rcvBearerAuth: []
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Poll' # TODO: Could also create a slim object to be returned here
+        '400':
+          description: Invalid Poll Details supplied
+          content: {}
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError' 
+        '403':
+          description: Poll Could Not be Updated (Already has a vote?) # TODO: Should updating a poll with a vote be forbidden or just clear the existing votes?
+          content: {}
+        '404':
+          description: Poll not found
+          content: {}
+
+    delete:
+      tags:
+        - Polls
+      summary: Deletes an existing Poll
+      operationId: deletePoll
+      parameters:
+        - name: pollID
+          in: path
+          description: Poll ID to interact with
+          required: true
+          schema:
+            type: integer
+      security:
+        - rcvBearerAuth: []
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Poll' # TODO: Could also create a slim object to be returned here
+        '400':
+          description: Invalid PollID supplied
+          content: {}
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError' 
+        '404':
+          description: Poll not found
+          content: {}
+
+  '/poll/{pollID}/results':
+    get:
+      tags:
+        - Polls
+      summary: Gets the Results of a specific Poll by its ID
+      operationId: getPollResults
+      parameters:
+        - name: pollID
+          in: path
+          description: Poll ID to interact with
+          required: true
+          schema:
+            type: integer
+      security:
+        - rcvBearerAuth: []
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PollResults'
+        '400':
+          description: Invalid PollID supplied
+          content: {}
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError' 
+        '404':
+          description: Poll not found
+          content: {}
+
+  '/poll/{pollID}/vote': #TODO: Probably need some header info about who the user placing the vote is

```

Comment:
Agreed! Would we want to do this in the request body in the VoteInput schema?
Will we want to add a user for Poll creation as well?

Future consideration: We could create/use a cookied JWT Token to verify who the user is if we don't want to do user management in the db.